status_deprecated.go

```diff
@@ -2,206 +2,116 @@ package ghostferry
 
```

Comment:
This file is still called `status_deprecated.go`?

Reply:
Moved to the code to fetch status to control_server.go so this file should be removed.